src/index.tsx

```diff
@@ -173,6 +177,22 @@ const setApplicationToStore = async (application?: Application) => {
     // @ts-ignore
     store.dispatch(setLogoPath(application.clientConfig.theme.logoPath));
   }
+
+  if (application.toolConfig) {
+    const availableTools: string[] = [];
+    application.toolConfig
+      // @ts-ignore
```

Comment:
Why do we need the `@ts-ignore`?

Reply:
yes, at least while this change is not merged and available.

https://github.com/terrestris/shogun-util/pull/88